app/bundles/CampaignBundle/Command/ResumeStuckCampaignCommand.php

```diff
@@ -0,0 +1,283 @@
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class ResumeStuckCampaignCommand extends Command
+{
+    use WriteCountTrait;
+
+    public const COMMAND_NAME = 'mautic:campaigns:resume-stuck';
+
+    private const MAX_ALLOWED_RECORDS_EACH_PROCESS = 10000;
+
+    public function __construct(
+        private TranslatorInterface $translator,
+        private EventModel $eventModel,
+        private CampaignModel $campaignModel,
+        private LeadModel $leadModel,
+        private EventExecutioner $eventExecutioner,
+        private ProcessSignalService $processSignalService,
+        private EventDispatcherInterface $eventDispatcher,
+    ) {
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setName(self::COMMAND_NAME)
+            ->setDescription('Resume execution for contacts stuck in campaign workflows.')
+            ->addArgument(
+                'campaign-id',
+                InputArgument::REQUIRED,
+                'The ID of the campaign to resume.'
+            )
+            ->addOption(
+                '--dry-run',
+                null,
+                InputOption::VALUE_NONE,
+                'Find next events without executing them.'
+            )
+            ->addOption(
+                '--batch-limit',
+                '-l',
+                InputOption::VALUE_OPTIONAL,
+                'Set batch size of contacts to process per round. Defaults to 100.',
+                1000
+            );
+
+        parent::configure();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $campaignId    = (int) $input->getArgument('campaign-id');
+        $batchLimit    = (int) $input->getOption('batch-limit');
+        $isDryRun      = (bool) $input->getOption('dry-run');
+        $this->processSignalService->registerSignalHandler(
+            fn (int $signal) => $output->writeln(sprintf('Signal %d caught.', $signal))
+        );
+
+        try {
+            $campaign = $this->campaignModel->getEntity($campaignId);
+            if (!$campaign) {
+                $output->writeln('<error>Campaign with ID '.$campaignId.' not found.</error>');
+
+                return ExitCode::FAILURE;
+            }
+
+            if (!$campaign->isPublished() || $campaign->isDeleted()) {
+                $output->writeln('<error>Campaign with ID '.$campaignId.' is not published or deleted.</error>');
+
+                return ExitCode::FAILURE;
```

Comment:
Please use Symfony's constants instead of custom ones in commands. There are more occurrences in this file.
```suggestion
                return Command::FAILURE;
```

Reply:
used directly from Command class except the termination signal as that is not present there.